animation_nodes/nodes/rotation/c_utils.pyx

```diff
@@ -1,9 +1,10 @@
-from libc.math cimport M_PI as PI
+from libc.math cimport M_PI as PI,sqrt
```

Comment:
Space after the comma. But I guess this will be removed anyway since it won't be needed.

Reply:
sorry for so many glitches in my code!